aws/ami/scylla.json

```diff
@@ -76,6 +76,7 @@
     },
     {
       "inline": [
+        "sudo /usr/bin/dnf -y install python2",
```

Comment:
Why do we need python2  for cqlsh ?

Reply:
scylla_install_ami is implemented in python2, since CentOS7 always has it, and only available version of python on the distribution.
It probably can work on python3 as well, but I don't find python2/3 in CentOS8 AMI, so we need to install prior to running the script.
Otherwise it just fail.
We may able to convert it to bash if python is not installed by default, though.

Reply:
I just verified it also works with python3.

Reply:
so this can go away ?